lib/mixlib/install/generator/bourne/scripts/platform_detection.sh

```diff
@@ -68,18 +68,22 @@ elif test -f "/etc/system-release"; then
     . /etc/os-release
     platform_version=$VERSION_ID
 
-    if test "$platform_version" = "2022"; then
-      platform="amazon"
-      platform_version="2022"
-    elif test "$platform_version" = "2"; then
-      platform="el"
-      platform_version="7"
-    else
-      platform="el"
-
-      # VERSION_ID will match YYYY.MM for Amazon Linux AMIs
-      platform_version="6"
-    fi
+    case $platform_version in
+      "2022"|"2023")
+        platform="amazon"
+        platform_version=$platform_version
```

Comment:
Not sure why this was being redefined in the original script like this, but I kept the same behavior just in case.

Reply:
@nejch 2022 version is replaced by the 2023 version, you can eliminate AL2022